internal/dkg/store.go

```diff
@@ -240,18 +198,10 @@ func (s *BoltStore) MigrateFromGroupfile(beaconID string, groupFile *key.Group,
 		Rejectors:     nil,
 		FinalGroup:    groupFile,
 		KeyShare:      share,
-	}
-
-	return s.SaveFinished(beaconID, &state)
+	}, nil
 }
 
-func (s *BoltStore) NukeState(beaconID string) error {
-	return s.db.Update(func(tx *bolt.Tx) error {
-		err := tx.Bucket(stagedStateBucket).Delete([]byte(beaconID))
-		if err != nil {
-			return err
-		}
-
-		return tx.Bucket(finishedStateBucket).Delete([]byte(beaconID))
-	})
+// NukeState deletes the directory corresponding to the specified beaconID
+func (fs *FileStore) NukeState(beaconID string) error {
+	return os.RemoveAll(fs.getDKGFolder(beaconID))
```

Comment:
I wonder if just renaming it wouldn't be better than actually nuking it? To keep the data around for further investigation

Reply:
We could probably even remove this now; I added this in the bolt version because deleting state for an individual `beaconID` was impossible without installing `boltbrowser` etc. Now that each `beaconID`'s DKG state is in a separate file, we can just `rm -rf` the relevant file

Reply:
IMO things managing the storage are better invoked through drand's cli.

How often are dkg stores nuked in practice?

Reply:
They shouldn't really ever be, only in the case of an unexpected error state that's not easily recoverable.

Reply:
I'm fairly confident I was having flaky test failures prior to removing the mutex, but it's worth being sure. I'll double check this.